tests/wpt/tests/html/semantics/forms/constraints/radio-group-valueMissing.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>
+  The constraint validation API Test: element.validity.valueMissing for radio
+  group
+</title>
+<link rel="author" title="Intel" href="http://www.intel.com/" />
```

Comment:
You should either remove this line or change it to add your name and email, like https://github.com/web-platform-tests/wpt/blob/5ff2643aa132c76bb9506a3becb59e2686fd34ec/css/css-lists/counter-slot-order.html#L5